examples/src/views/new-example.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'views/templates/document.html' %}
+
+{% import 'components/Document/Head.html' as Head %}
+{% import 'components/Sidebar/Sidebar.html' as Sidebar %}
+{% import 'components/PageHeader/PageHeader.html' as PageHeader %}
+
+{% import 'content/Draggable/NewExample/NewExample.html' as NewExample %}
+
+{% set ViewAttrs = {
+  id: 'NewExample',
+  parent: 'Draggable',
+  child: 'New example',
+  subheading: 'This is the description I was asked to enter during the "new page" generator that Promhize built.'
+} %}
+
+{% block PageId %}{{ ViewAttrs.id }}{% endblock %}
+
+{% block head %}
+  {{ Head.render(ViewAttrs) }}
+{% endblock %}
+
+{% block sidebar %}
+  {{ Sidebar.render(ViewAttrs, DataPages) }}
+{% endblock %}
+
+{% block main %}
+  {{ PageHeader.render(ViewAttrs.parent, ViewAttrs.subheading) }}
+  {{ NewExample.render(ViewAttrs.id) }}
+{% endblock %}
```

Comment:
Just to add a bit more clarity to this file... take a look at the snippet below to see the HTML comments indicating where something is referencing the "new example" being created.

```html
{% extends 'views/templates/document.html' %}

{% import 'components/Document/Head.html' as Head %}
{% import 'components/Sidebar/Sidebar.html' as Sidebar %}
{% import 'components/PageHeader/PageHeader.html' as PageHeader %}

<!-- IMPORT THE NEW EXAMPLE CONTENT MACRO -->
{% import 'content/Draggable/NewExample/NewExample.html' as NewExample %}

<!-- BUILD THE NEW EXAMPLE ViewAttrs OBJECT -->
{% set ViewAttrs = {
  id: 'NewExample',
  parent: 'Draggable',
  child: 'New example',
  subheading: 'This is the description I was asked to enter during the "new page" generator that Promhize built.'
} %}

{% block PageId %}{{ ViewAttrs.id }}{% endblock %}

{% block head %}
  {{ Head.render(ViewAttrs) }}
{% endblock %}

{% block sidebar %}
  {{ Sidebar.render(ViewAttrs, DataPages) }}
{% endblock %}

{% block main %}
  {{ PageHeader.render(ViewAttrs.parent, ViewAttrs.subheading) }}
  <!-- RENDER THE NEW EXAMPLE CONTENT MACRO -->
  {{ NewExample.render(ViewAttrs.id) }}
{% endblock %}
```